pyscriptjs/tests/integration/test_zz_examples.py

```diff
@@ -360,15 +360,31 @@ def test_todo(self):
         self.assert_no_banners()
         self.check_tutor_generated_code(modules_to_check=["./utils.py", "./todo.py"])
 
-    @pytest.mark.xfail(reason="fails after introducing synclink, fix me soon!")
     def test_todo_pylist(self):
-        # XXX improve this test
         self.goto("examples/todo-pylist.html")
         self.wait_for_pyscript()
         assert self.page.title() == "Todo App"
         wait_for_render(self.page, "*", "<input.*?id=['\"]new-task-content['\"].*?>")
+        todo_input = self.page.locator("input")
```

Comment:
Can you add comments in here in the form or "when we type bla bla in the input element..." --> "assert bla bla"

It's not a merge blocker but I'd like for us to improve the readability of tests...